### PR TITLE
Converting Table to Grid of Cards

### LINKED ISSUE #1316: Replace Tables with Grid of Cards
Tables aren't intuitively easy to read information from, so I want to propose a design change to using a Grid of Cards.
The design still uses the `table` element in the back end, so the filtering still functions exactly the same.

It primarily would use `.tbl-gridify` built into wet, and would require a small amount of custom styling.

_The design actually comes from the WET-BOEW styleguid https://wet-boew.github.io/wet-boew-styleguide/index-en.html (they are using a table element)_

image

### PR DESCRIPTION
**Pull request by s-laugh**
This fixes #1316 

Wanted to get a review with one table modified for any suggestions before changes the others.

### CODE REVIEW

**Comment by s-laugh** [comment]
@nschonni Looking for a review on this, ensuring I can continue with this implementation on other tables.

**Comment by s-laugh** [comment]
@nschonni @gcharest @smellems Any able to review? - provide further thoughts on the approach?

**Comment by smellems** [comment]
I think this is great!  I will reuse this design for the micro-acquisition pilot site we are working on.

I noticed that HTML validation show warnings for: `role="row"` is unnecessary for element tr.  I tried removing it and it looks like it still works.

Not sure if I should commit to your branch, but so that every page is filled to the max, you can use `"lengthMenu": [12,24,48,96]`
https://datatables.net/reference/option/lengthMenu

**Comment by s-laugh** [comment]
Awesome! thanks, I will make those adjustments!

**Comment by s-laugh** [comment]
I'm also fine if you push to my branch (if you're not being blocked)

File: _includes/codeTable.html [hunk 1]
```diff
@@ -196,22 +201,22 @@
...
     </thead>
-    <tbody>
+    <tbody class="row wb-eqht">
       {%- for govLevels in site.data.code -%}
         {%- assign govLevel = govLevels[0] -%}
         {%- for admin in govLevels[1] -%}
           {%- for release in admin[1].releases -%}
           {%- unless release.tags.en contains site.excludeTag -%}
-          <tr>
+          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;" role="row">
```
Comment (nschonni) [inline _includes/codeTable.html:210]: Think this is from @smellems comment
```suggestion
          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;">
```

**Review by nschonni — APPROVED** [review]
Haven't had a change to play with this locally yet, but lets land and try it out

**Pull request merged**